docs/pages/radiogroup.js

```diff
@@ -0,0 +1,986 @@
+                      <IconButton size="sm" icon="info-circle" iconColor="darkGray" accessibilityLabel="info" tooltip={{text: "Sets a cap for the amount your campaign can spend over the course of its lifetime"}}/>
+                    </Flex>
+                  </Label>
+                </Flex>
+              </RadioGroup>
+          );
+        }
+                `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Legend visibility"
+          description={`
+            By default, the \`legend\` is visible above the items in the RadioGroup. However, if the form items are labeled by content elsewhere on the page, or a more complex legend is needed, the \`legendDisplay\` prop can be used to visually hide the \`legend\`. In this case, it is still available to screen reader users, but will not appear visually on the screen.
+
+            In the example below, the "Primary company account goal" text is acting as a heading and a legend for the radio buttons, so instead of repeating another legend, we visually hide the RadioGroup \`legend\`. When a user focuses on the first radio, a screen reader will announce "Sell more products, radio button, 1 of 3, Primary company account goal, group".
+        `}
+        >
+          <MainSection.Card
+            defaultCode={`
+function RadioButtonExample() {
+  const [goal, setGoal] = React.useState();
+
+  return (
+    <Flex direction="column" gap={4}>
+      <Flex direction="column" gap={2}>
+        <Heading size="400">Primary company account goal</Heading>
+        <Text size="200">
+          Choose your primary goal for this account to help us better understand your needs
+          <Text inline size="200" weight="bold">
+            <Link inline target="blank" href="https://www.pinterest.com/">
+              Additional information
+            </Link>
+          </Text>
+        </Text>
+      </Flex>
+      <RadioGroup legend="Primary company account goal" legendDisplay="hidden" id="legendExample">
+        <RadioGroup.RadioButton
+          checked={goal === "sell"}
+          id="sell"
+          label="Sell more products"
+          name="account goals"
+          onChange={() => setGoal('sell')}
+          value="sell"
+        />
+        <RadioGroup.RadioButton
+          checked={goal === "leads"}
+          id="leads"
+          label="Generate more leads for the company"
+          name="account goals"
+          onChange={() => setGoal('leads')}
+          value="leads"
+        />
+        <RadioGroup.RadioButton
+          checked={goal === "interest"}
+          id="interest"
+          label="Create content on Pinterest to attract an audience"
+          name="account goals"
+          onChange={() => setGoal('interest')}
+          value="interest"
+        />
+      </RadioGroup>
+    </Flex>
+  );
+}
+        `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
```

Comment:
I didn't include this sectoin when we rewrote Checkbox. Thoughts about removing it?

Reply:
I'm fine with removing it, since it's not a prop we encourage people to use unless their use case really, really calls for it